ts/components/dialog/edit-profile/EditProfileDialog.tsx

```diff
@@ -226,11 +212,17 @@ export const EditProfileDialog = () => {
       return;
     }
 
-    setLoading(true);
-    await updateDisplayName(profileName);
-    setUpdateProfileName(profileName);
-    setMode('default');
-    setLoading(false);
+    try {
+      setLoading(true);
+      const validName = await ProfileManager.updateOurProfileDisplayName(profileName);
+      setUpdateProfileName(validName);
+      setMode('default');
+    } catch (err) {
+      window.log.error('Profile update error', err);
+      setProfileNameError(window.i18n('unknownError'));
+    } finally {
+      setLoading(false);
+    }
```

Comment:
is the error found by gh action here a false positive?

![image](https://github.com/user-attachments/assets/bfb6ae7c-37d5-4f31-86ca-7e98d07f25b7)

Reply:
don't see an issue so I guess we are fine